internal/cmdsearch/cmdsearch.go

```diff
@@ -86,21 +90,34 @@ func (r *SearchRunner) preRunE(c *cobra.Command, args []string) error {
 		c.Flag("by-value-regex").Changed {
 		return errors.Errorf(`only one of ["by-value", "by-value-regex"] can be provided`)
 	}
+	r.Writer = c.OutOrStdout()
```

Comment:
Why is there both a writer on the `ExecuteCmdOnPkgs` and on the `SearchRunner`? It is confusing.

Reply:
I realized that `ExecuteCmdOnPkgs` should not write anything. The Runners of commands should only be responsible to write. I will refactor the cmd/config accordingly so that this will be clear.